doc/source/api.rst

```diff
@@ -289,9 +305,15 @@ Conversion
 
    Series.astype
    Series.infer_objects
```

Comment:
deprecated does it still have to be here? (maybe separate section)?

Reply:
I wasn't sure about that. Since these are the docs for this version, deprecated methods should be somewhere. Happy to move it if needed.

Reply:
I think you have a section already for deprecated things